Jolt/Core/Core.h

```diff
@@ -191,6 +191,16 @@
 		#define JPH_USE_SSE4_1
 		#define JPH_USE_SSE4_2
 	#endif
+#elif defined(__ppc64__) || defined(_M_PPC64) || defined(__powerpc__) || defined(_M_PPC)
```

Comment:
I would expect many more code changes to be needed. As far as I understand PPC is a big endian architecture and Jolt makes quite a few assumptions about it being little endian. I don't really want to have to take endian swapping into account everywhere as it results in really messy code. Did you try running the unit tests? 

I would add a build here: https://github.com/jrouwe/JoltPhysics/blob/master/.github/workflows/build.yml which runs the unit tests using `qemu-powerpc` similar to https://github.com/jrouwe/JoltPhysics/blob/e0e434f59c2ff6816e07af7f6f6fcf9726e2aa0f/.github/workflows/determinism_check.yml#L154-L171